### PR TITLE
add docs for track user data

### PR DESCRIPTION
**Pull request by topi314**

### CODE REVIEW

File: docs/api/rest.md [hunk 1]
```diff
@@ -53,11 +53,12 @@ When Lavalink encounters an error, it will respond with a JSON object containing
...
-| Field      | Type                             | Description                             |
-|------------|----------------------------------|-----------------------------------------|
-| encoded    | string                           | The base64 encoded track data           |
-| info       | [Track Info](#track-info) object | Info about the track                    |
-| pluginInfo | object                           | Addition track info provided by plugins |
+| Field      | Type                             | Description                                                                   |
+|------------|----------------------------------|-------------------------------------------------------------------------------|
+| encoded    | string                           | The base64 encoded track data                                                 |
+| info       | [Track Info](#track-info) object | Info about the track                                                          |
+| pluginInfo | object                           | Addition track info provided by plugins                                       |
```
Comment (duncte123) [inline docs/api/rest.md]: This feels like a typo

```suggestion
| pluginInfo | object                           | Additional track info provided by plugins                                       |
```

Comment (duncte123) [inline docs/api/rest.md]: This feels like a typo

```suggestion
| pluginInfo | object                           | Additional track info provided by plugins                                       |
```

File: docs/api/rest.md [hunk 2]
```diff
@@ -53,11 +53,12 @@ When Lavalink encounters an error, it will respond with a JSON object containing
...
+| userData   | object                           | Addition track data provided via the [Update Player](#update-player) endpoint |
```
Comment (duncte123) [inline docs/api/rest.md]:
```suggestion
| userData   | object                           | Additional track data provided via the [Update Player](#update-player) endpoint |
```

Comment (duncte123) [inline docs/api/rest.md]:
```suggestion
| userData   | object                           | Additional track data provided via the [Update Player](#update-player) endpoint |
```

**Review by duncte123 — CHANGES_REQUESTED** [review]

File: docs/api/rest.md [hunk 3]
```diff
@@ -684,20 +693,26 @@ Query Params:
 
 Request:
 
-| Field           | Type                               | Description                                                                                   |
-|-----------------|------------------------------------|-----------------------------------------------------------------------------------------------|
-| encodedTrack? * | ?string                            | The base64 encoded track to play. `null` stops the current track                              |
-| identifier? *   | string                             | The identifier of the track to play                                                           |
```
Comment (freyacodes) [inline docs/api/rest.md:690]: These are deprecated, but I think it is still important to mention them

File: docs/api/rest.md [hunk 4]
```diff
@@ -684,20 +693,26 @@ Query Params:
...
-| volume?         | int                                | The player volume, in percentage, from 0 to 1000                                              |
-| paused?         | bool                               | Whether the player is paused                                                                  |
-| filters?        | [Filters](#filters) object         | The new filters to apply. This will override all previously applied filters                   |                   
-| voice?          | [Voice State](#voice-state) object | Information required for connecting to Discord                                                |
+| Field         | Type                                        | Description                                                                                   |
+|---------------|---------------------------------------------|-----------------------------------------------------------------------------------------------|
+| track?        | [Update Player Track](#update-player-track) | The `encoded` track, `identifier` and `userData` of the track to play                         |
```
Comment (freyacodes) [inline docs/api/rest.md]:
```suggestion
| track?        | [Update Player Track](#update-player-track) | Specification for a new track to load, as well as user data to set                       |
```

**Review by freyacodes — CHANGES_REQUESTED** [review]

File: docs/api/rest.md (hunk 3 above)
Comment (duncte123) [reply docs/api/rest.md:690]: I agree with Freya here

Comment (topi314) [reply docs/api/rest.md:690]: done
image

**Pull request merged**